packages/shared/src/components/squads/Details.tsx

```diff
@@ -6,25 +6,27 @@ import { Button } from '../buttons/Button';
 import { TextField } from '../fields/TextField';
 import AtIcon from '../icons/At';
 import Textarea from '../fields/Textarea';
-import { SquadSubTitle, SquadTitle } from './utils';
 import ImageInput from '../fields/ImageInput';
 import { cloudinary } from '../../lib/image';
 import CameraIcon from '../icons/Camera';
 import { formToJson } from '../../lib/form';
-import { Modal } from '../modals/common/Modal';
 import { blobToBase64 } from '../../lib/blob';
 import { checkExistingHandle, SquadForm } from '../../graphql/squads';
 import { capitalize } from '../../lib/strings';
 import { IconSize } from '../Icon';
-import { Justify } from '../utilities';
+import { FlexCol } from '../utilities';
```

Comment:
I'm not sure we should rely on these for now.
It becomes dangerous when someone decides to add a class to this FlexCol for one bug and forgets to check other places.
It's more a generic comment for now until we can support a full blown component library maybe?

@sshanzel What do you think?

Reply:
I mean we did encourage using those in the past. If you ask me we don't even need those util components but happy with whatever you decide 👍

Reply:
To me they propose more danger then benefit anything from them 😅